cli/upload.py

```diff
@@ -243,10 +244,15 @@ def _compose_file_mapping(upload_info: api.UploadInfo, xlsx: str):
             source_path = os.path.join(xlsx_dir, source_path)
 
             if not os.path.isfile(source_path):
-                raise Exception(f"Couldn't locate file {source_path}")
+                missing_files.append(source_path)
 
         res.append([source_path, f"gs://{upload_info.gcs_bucket}/{gcs_uri}"])
 
+    if missing_files:
+        raise Exception(
+            f'Could not locate files: {", ".join(str(x) for x in missing_files)}'
```

Comment:
Since `missing_files` is already a list of file path strings, I don't think this conversion is necessary, so
```python
", ".join(missing_files)
```
should do the trick.